### DISCUSSION TITLE
Dedicated Page for Multi-part Posts with Nested Routing

### DISCUSSION THREAD

**Discussion by khannzxc** [timlrx/tailwind-nextjs-starter-blog#446] [Q&A] [answered]
I would be nice to have a **Dedicated Page for Multi-part Posts with Nested Routing**. For example:
`https://tailwind-nextjs-starter-blog.vercel.app/blog/nested-route/*` will show **page** showing a list of all posts related to `nested-route` categories as [in this jekyll theme](https://chirpy.cotes.page/categories/blogging/).
In case of this theme [this link](https://tailwind-nextjs-starter-blog.vercel.app/blog/nested-route/) shows 404 page.
Thanks

**Comment by dlarroder** [comment, marked as answer]
This is a cool idea but this template already supports tags, if you want to group your multi part posts, you can just add the same tags to the different posts and the user can click the tags and it will group them by tag. for example: https://tailwind-nextjs-starter-blog.vercel.app/tags/next-js
